src/tests/Common/XUnitWrapperLibrary/TestSummary.cs

```diff
@@ -3,40 +3,127 @@
 //
 
 using System;
+using System.IO;
 using System.Collections.Generic;
 using System.Text;
 namespace XUnitWrapperLibrary;
 
 public class TestSummary
 {
-    readonly record struct TestResult(string Name, string ContainingTypeName, string MethodName, TimeSpan Duration, Exception? Exception, string? SkipReason, string? Output);
+    // readonly record struct TestResult(string Name, string ContainingTypeName, string MethodName, TimeSpan Duration, Exception? Exception, string? SkipReason, string? Output);
```

Comment:
Probably obsolete comment worth deleting. Just for my education, what is the benefit of using <code>record</code> here, is that about something like automatically generated <code>IComparer / Equals / GetHashCode</code> implementations or is there more to it?

Reply:
I'm not sure there is a tangible benefit/degrade to using record here. It was already there so I opted to not touch it since I was able to adapt it easily :)